src/D2L.Bmx/ConfigureHandler.cs

```diff
@@ -28,7 +29,9 @@ internal class ConfigureHandler {
 			defaultDuration = _consolePrompter.PromptDefaultDuration();
 		}
 
-		bool allowProjectConfigs = _consolePrompter.PromptAllowProjectConfig();
+		if( !allowProjectConfigs ) {
+			allowProjectConfigs = _consolePrompter.PromptAllowProjectConfig();
```

Comment:
What if user wants to explicitly specify `--allow-project-configs false` on the CLI and not be prompted here?
I think we need to make `allowProjectConfigs` a `bool?` to allow three states: `null`, `true`, `false`.

Reply:
Yea I originally tried this but didn't change the type and I think that's where my issue was. So changing to `bool?` will fix that